src/ucp/core/ucp_mm.c

```diff
@@ -272,6 +276,10 @@ static inline ucs_status_t ucp_mem_map_check_and_adjust_params(ucp_mem_map_param
                (params->flags & UCP_MEM_MAP_FIXED)) {
         ucs_error("Wrong combination of flags when address is defined");
         return UCS_ERR_INVALID_PARAM;
+    } else if ((params->flags & UCP_MEM_MAP_DEVICE_MEM) &&
+               !(params->flags & UCP_MEM_MAP_ALLOCATE)) {
+        ucs_error("Wrong combination of flags when DM requested and address defined");
```

Comment:
ucs_error("Wrong combination of flags when DM requested and allocate flag not specified");